tests/utils/coverage_subprocess_bootstrap.py

```
@@ -0,0 +1,46 @@
+
+    coverage.process_startup()
+    _cov = getattr(coverage.process_startup, "coverage", None)
+    if _cov is not None and os.environ.get("COVERAGE_PROCESS_START"):
+
+        def _periodic_save():
+            while True:
+                time.sleep(_SAVE_INTERVAL)
+                try:
+                    _cov.save()
+                except Exception:
+                    pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)